config/helm/delete-deployments.yaml

```diff
@@ -0,0 +1,32 @@
+# In our current v0.3.x release we are deploying only webhooks, so we do not
```

Comment:
Patch to delete controller deployments, because in this release (0.3.x) we deploy only webhooks.